test/test_pipeline/components/data_preprocessing/test_data_preprocessing_categorical.py

```diff
@@ -10,20 +10,26 @@ class CategoricalPreprocessingPipelineTest(unittest.TestCase):
 
     def test_fit_transform(self):
         X = np.array([
-            [1, 2, 0],
-            [3, 0, 0],
+            [1, 2, 1],
+            [3, 1, 1],
             [2, 9, np.nan]])
         Y = np.array([
             [1, 0, 0, 0, 1, 0, 0, 1],
             [0, 0, 1, 1, 0, 0, 0, 1],
             [0, 1, 0, 0, 0, 1, 1, 0]])
         # dense input
         Yt = CategoricalPreprocessingPipeline().fit_transform(X)
-        self.assertTrue((Yt.todense() == Y).all())
+        Yt = Yt.todense()
+        self.assertTrue(Yt.shape == Y.shape)
```

Comment:
Please use `assertEqual` here

Reply:
Done.

Reply:
I deleted this line and used **numpy.testing.assert_array_equal()** instead, since it also checks the shape.